test/index.html

```diff
@@ -71,12 +71,12 @@
     top.on(message => {
       console.log('[hub.top] message %o', message);
     });
-    setTimeout(function() {
-      top.publish('Tradeshift.ForeignSandbox', 'heyo1', 'yea');
-      top.publish('Tradeshift.Foreign', 'hey2o', 'yea');
-      top.publish('Tradeshift.SameSandbox', 'heyo3', 'yea');
-      top.publish('Tradeshift.Same', 'hey4o', 'yea');
-    }, 1000);
+    // setTimeout(function() {
+    //   top.publish('Tradeshift.ForeignSandbox', 'heyo1', 'yea');
+    //   top.publish('Tradeshift.Foreign', 'hey2o', 'yea');
+    //   top.publish('Tradeshift.SameSandbox', 'heyo3', 'yea');
+    //   top.publish('Tradeshift.Same', 'hey4o', 'yea');
+    // }, 1000);
```

Comment:
Is this broken now? Or why is it commented out?

Reply:
oh yea, I commented it out so I clearly see what's happening when looking at debug messages. will uncomment.